index.html

```diff
@@ -0,0 +1,286 @@
+                    </ul>
+                  </div>
+               </li>
+                <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio3.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Facebook 360</a></h1>
+                        <div class="work-info">
+                          <ul>
+                            <li class="intro">FACEBOOK</li>
+                            <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                            <li><i class="fa fa-circle"></i> 2015</li>
+                          </ul>
+                          </div>
+                        <p>
+                       Exploring the future of media in Facebook's first 
+                       Virtual Reality app; a place to
+                        discover and enjoy 360 photos and videos on Gear VR.</p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-two">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+
+                <li class="work">            
+                  <div class="work-container">
+                    <ul>
+                      <li>
+                          <img src="image/portfolio4.png" alt="Several hands holding beer glasses">
+                      <li class="work-description">
+                        <h1><a href="">Uber Navigation</a></h1>
+                        <div class="work-info">
+                          <ul>
+                            <li class="intro">UBER</li>
+                            <li> <i class="fa fa-circle"></i> Lead Developer</li>
+                            <li><i class="fa fa-circle"></i> 2018</li>
+                          </ul>
+                        </div>
+                        <p>
+                        A smart assistant to make driving more safe, 
+                        efficient, and fun by unlocking 
+                        your most expensive computer: your car.</p>
+
+                        <div class="tags">
+                          <ul>
+                            <li>html</li>
+                            <li>Ruby on Rails</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                          </ul>
+                        </div>
+
+                        <a href="#" class="btn-one">See project</a>
+                      </li>
+                    </ul>
+                  </div>
+               </li>
+          </ul>
+  </section>
```

Comment:
- [ ] Please add hover color to the `see project` button and add some padding between the project technology.